test/gtest/uct/ib/test_ib_pkey.cc

```diff
@@ -182,7 +185,10 @@ UCS_TEST_P(test_uct_ib_pkey, test_pkey_pairs) {
         m_e2 = uct_test::create_entity(0);
         m_entities.push_back(m_e2);
 
-        prepare_entities();
+        m_e1->connect(0, *m_e2, 0);
+        m_e2->connect(0, *m_e1, 0);
+
+        prepare_entities(false);
```

Comment:
prepare_entities(false) only does couple of EXPECT_EQ checks
maybe they could go to separate helper func, instead of passing true/false?

Reply:
yes, good catch
it would be better
done